dags/resources/stages/parse/table_definitions/damm/JumpRateModelV2_event_NewInterestParams.json

```diff
@@ -0,0 +1,65 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "baseRatePerBlock",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "multiplierPerBlock",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "jumpMultiplierPerBlock",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "kink",
+                    "type": "uint256"
+                }
+            ],
+            "name": "NewInterestParams",
+            "type": "event"
+        },
+        "contract_address": "0x4f96ab61520a6636331a48a11eafba8fb51f74e4",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "damm",
+        "schema": [
+            {
+                "description": "",
+                "name": "baseRatePerBlock",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "multiplierPerBlock",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "jumpMultiplierPerBlock",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "kink",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "JumpRateModelV2_event_NewInterestParams"
```

Comment:
The address is the same as ComptrollerG7, which does not seem to implement this event. Either the address or the name seems incorrect:)

Reply:
It seems like ComptrollerG7 was turned into Unicontroller - https://etherscan.io/tx/0xafd09fefeeca521bcb4d14b9da8a1eac998aee761dec64cab89602b40969e738

will remove the ComptrollerG7 files

Reply:
removed this file - it shouldn't have existed in the first place